pkg/cmd/cli/nodeagent/server.go

```diff
@@ -200,13 +200,31 @@ func newNodeAgentServer(logger logrus.FieldLogger, factory client.Factory, confi
 			},
 		},
 	}
-	mgr, err := ctrl.NewManager(clientConfig, ctrl.Options{
-		Scheme: scheme,
-		Cache:  cacheOption,
-	})
+
+	var mgr manager.Manager
```

Comment:
This is to resolve the problem that in some Windows env, the pod network is not immediately ready. Without this, the pod may restart for several times